lib/vdsm/virt/vmdevices/storage.py

```diff
@@ -1125,6 +1125,8 @@ def _getDriverXML(drive):
 
     if 'discard' in drive and drive['discard']:
         driverAttrs['discard'] = 'unmap'
+        if 'discard_no_unref' in drive and drive['discard_no_unref']:
+            driverAttrs['discard_no_unref'] = 'on'
```

Comment:
This we can better do
```
if 'discard_no_unref' in drive:
        driverAttrs['discard_no_unref'] = drive['discard_no_unref']
```

?
Then it will just take the argument we've got from the engine as argument to discard_no_unref.